src/test/java/uk/gov/hmcts/ccd/endpoint/userprofile/UserProfileEndpointIT.java

```diff
@@ -210,10 +210,9 @@ public void createExistingUserProfile() throws Exception {
                         .content(mapper.writeValueAsBytes(userProfile)))
                 .andReturn();
 
-        assertEquals("Unexpected response status", 400, mvcResult.getResponse().getStatus());
-        assertEquals("Unexpected response message",
-                     "User already exists with Id " + USER_ID_1,
-                     mvcResult.getResponse().getContentAsString());
+        assertEquals(201, mvcResult.getResponse().getStatus());
```

Comment:
if we want to go with the ignore solution, I think we should not return 201 in this case. Can we return a more suitable response code and message to indicate that the request was ignored?